config/locales/en.yml

```diff
@@ -153,6 +153,7 @@ en:
     appeal_to_crown_court: Appeal to crown court
     appeal_to_crown_court_with_changes: Appeal to crown court with changes in financial circumstances
     class: Class
+    class_not_determined: "CLASS\xA0NOT\xA0DETERMINED"
```

Comment:
I think no need to be all uppercase, the css styling does that for you.
The escape sequence I presume is to try to keep it all in one line? Unsure how that may work on small screens but probably we can assume all case workers will use desktop screens 👍